src/philips-hue.coffee

```diff
@@ -202,40 +218,25 @@ module.exports = (robot) ->
             if (line.success)
               msg.send "Hash:" + line.success.username
 
-  robot.respond /hue set config ([^\s]*) (.*)/i, (msg) ->
-    [setting,val] = msg.match[1..2]
-    supportedSettings = ['name','linkbutton']
-    if setting in supportedSettings
-
-      switch setting
-        when "name" then jsonParams = name: val
-        when "linkbutton" then jsonParams = linkbutton: val
-
-      msg.send JSON.stringify(jsonParams)
-      url = "http://#{base_url}/api/#{hash}/config"
-      setInfo msg, url, jsonParams, (responseText) ->
-        msg.send responseText
-    else
-      msg.send "unsupported config setting"
-
-
-#  robot.respond /rollout activate_user ([^\s]*) ([^\s]*)/i, (msg) ->
-#    msg.http(endpoint + msg.match[1] + '/users').query(user: msg.match[2]).put() (err, res, body) ->
-#      show(msg, msg.match[1])
-
-getGenInfo = (msg, url, callback) ->
-    msg.http(url)
-      .headers(Accept: 'application/json')
-      .get() (err, res, body) ->
-         if 200 <= res.statusCode <= 299
-           callback JSON.stringify(JSON.parse(body),null,'\t')
-         else
-           callback res.statusCode
-
-setInfo = (msg, url, jsonParams, callback) ->
-    msg.http(url)
-      .put(JSON.stringify(jsonParams)) (err, res, body) ->
-            if 200 <= res.statusCode <= 299
-              callback JSON.stringify(JSON.parse(body),null,'\t')
-            else
-              callback res.statusCode
+  robot.respond /hue (link|linkbutton|link button)/i, (msg) ->
```

Comment:
Yeah, in trying to conform with what the library provided (instead of direct queries), I could not find a command to set the Bridge name. In practice, I really don't think that would be terribly useful. However, triggering the link button might be for various reasons.